packages/apptive_grid_core/lib/src/model/entity/entities_response.dart

```diff
@@ -24,6 +46,68 @@ class EntitiesResponse<T> {
   EntitiesResponse copyWith({List<T>? items}) {
```

Comment:
The Page Meta Data should also be passed in

Reply:
done